website/docs/docs/core/connect-data-platform/athena-setup.md

```diff
@@ -4,7 +4,7 @@ description: "Read this guide to learn about the Athena warehouse setup in dbt."
 meta:
   maintained_by: dbt Labs
   authors: dbt Labs
-  github_repo: 'dbt-labs/dbt-athena'
+  github_repo: '/dbt-labs/dbt-adapters/tree/main/dbt-athena'
```

Comment:
```suggestion
  github_repo: 'dbt-labs/dbt-adapters'
```

Reply:
arg it's giving a 404 badge issue so going to keep it to adapters
![Screenshot 2025-03-11 at 17 16 18](https://github.com/user-attachments/assets/381cead4-8e8d-4b21-8bbb-e6e1cdfa56a9)